docs/guides/configure-acl-plugin.md

```diff
@@ -0,0 +1,694 @@
+
+HTTP/1.1 401 Unauthorized
+Date: Mon, 06 Apr 2020 07:27:44 GMT
+Content-Type: application/json; charset=utf-8
+Connection: keep-alive
+Content-Length: 50
+X-Kong-Response-Latency: 2
+Server: kong/2.0.2
+
+
+{"message":"Unauthorized"}
+```
+
+You should get a 401 response telling you that the request is not authorized.
+
+## Provision Consumers
+
+Let's provision 2 KongConsumer resources:
+
+```bash
+$ echo "
+apiVersion: configuration.konghq.com/v1
+kind: KongConsumer
+metadata:
+  name: admin
+username: admin
+" | kubectl apply -f -
+
+$ echo "
+apiVersion: configuration.konghq.com/v1
+kind: KongConsumer
+metadata:
+  name: plain-user
+username: plain-user
+" | kubectl apply -f -
+```
+
+## Secrets
+
+Next, let's provision some Secrets for the KongConsumers to reference. Each ACL will need it's own Secret and each JWT public key will need it's own Secret. The credential type is specified in the `kongCredType` field. In this case we'll be using `jwt` and `acl`. You can create a secret using any other method as well.
+
+The JWT signing algorithm is set in the `algorithm` field. The if using a public key like this example it is stored in the `rsa_pulic_key` field. If you are using a secret signing key, use the `secret` field. The last field to set if you are using `RS256` or `ES256` is the `key` field. This should match the `iss` field in the JWT you will be sending. You can check this value by decoding your JWT over at [https://jwt.io](https://jwt.io)
+
+Since we are using the Secret resource, Kubernetes will encrypt and store the JWT signing key and ACL group for us.
+
+### JWT signing key
+
+```bash
+# create secret for jwt public key
+$ kubectl create secret \
+  generic app-admin-jwt  \
+  --from-literal=kongCredType=jwt  \
+  --from-literal=key="admin-issuer" \
+  --from-literal=algorithm=RS256 \
+  --from-literal=rsa_public_key="-----BEGIN PUBLIC KEY-----\nMIIBIjA....-----END PUBLIC KEY-----"
+
+# create a second secret with a different key
+$ kubectl create secret \
+  generic app-user-jwt  \
+  --from-literal=kongCredType=jwt  \
+  --from-literal=key="user-issuer" \
+  --from-literal=algorithm=RS256 \
+  --from-literal=rsa_public_key="-----BEGIN PUBLIC KEY-----\qwerlkjqer....-----END PUBLIC KEY-----"
+```
+
+## Assign the credentials
+
+In order to for the ACL and JWT to be validated by Kong, the secrets will need to be referenced by the KongConsumers we created earlier. Let's update those.
+
+```bash
+$ echo "
+apiVersion: configuration.konghq.com/v1
+kind: KongConsumer
+metadata:
+  name: admin
+username: admin
+credentials:
+  - app-admin-jwt
+" | kubectl apply -f -
+
+$ echo "
+apiVersion: configuration.konghq.com/v1
+kind: KongConsumer
+metadata:
+  name: plain-user
+username: plain-user
+credentials:
+  - app-user-jwt
+" | kubectl apply -f -
+```
+
+## Use the credential
+
+Now to use a JWT to pass authentication. Let's store the user and admin jwt's in some environment variables. `USER_JWT` and `ADMIN_JWT`
```

Comment:
Can you please add the steps of values that are set in these variables?

Reply:
done